remo/settings/base.py

```diff
@@ -233,6 +233,7 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
 CSP_SCRIPT_SRC = (
     "'self'",
     "'unsafe-eval'",  # TODO: remove this by pre-compiling handlebars
+    'data:',
```

Comment:
Because mapbox is loading a data URI that is currently rejected by the CSP rules.